style_traits/lib.rs

```diff
@@ -33,11 +33,7 @@ extern crate to_shmem;
 #[macro_use]
 extern crate to_shmem_derive;
 #[cfg(feature = "servo")]
-extern crate webrender_api;
-#[cfg(feature = "servo")]
```

Comment:
Doesn't removing the webrender dependency here break the Servo compilation?

Reply:
Yes it does. We need to come to some kind of consensus on what the best solution to this is.

My thought is that it probably does make sense to decouple the two libraries by removing the Webrender dependency here and fixup Servo to deal with the types being different. The advantage of doing this is that:

- Servo would be able to update Stylo and Webrender versions independently of each other
- Crates.io releases of Stylo and Webrender could follow independent release schedules.

IIRC there were 9 errors when I tried compiling Servo with this branch so it's not a huge task. I would be able to do this work if there's consensus that this an acceptable approach.